packages/alfa-rules/src/sia-r61/rule.ts

```diff
@@ -0,0 +1,57 @@
+import { Rule } from "@siteimprove/alfa-act";
+import { Document, Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+
+import { expectation } from "../common/expectation";
+import { getAriaLevel } from "../common/expectation/get-aria-level";
+import { hasChild } from "../common/predicate/has-child";
+import { hasName } from "../common/predicate/has-name";
+import { hasRole } from "../common/predicate/has-role";
+import { isDocumentElement } from "../common/predicate/is-document-element";
+
+const { and, equals } = Predicate;
+
+export default Rule.Atomic.of<Page, Document>({
+  uri: "https://siteimprove.github.io/sanshikan/rules/sia-r61.html",
+  evaluate({ device, document }) {
+    let firstHeading: Element;
+
+    return {
+      applicability() {
+        if (hasChild(and(Element.isElement, isDocumentElement()))(document)) {
+          const myFirstHeading = document
+            .descendants({ flattened: true })
+            .filter(and(Element.isElement, hasRole(hasName(equals("heading")))))
+            .first();
+          if (myFirstHeading.isSome()) {
+            firstHeading = myFirstHeading.get();
+            return [document];
+          }
+        }
+        return [];
+      },
+
+      expectations(target) {
+        return {
+          1: expectation(
+            getAriaLevel(firstHeading, device).every((level) => level === 1),
```

Comment:
Is there a way to return browser specific outcomes instead of collapsing everything?